bindings/nim/kzg_abi.nim

```diff
@@ -86,32 +92,32 @@ proc load_trusted_setup_file*(res: KzgSettings,
 
 proc free_trusted_setup*(s: KzgSettings) {.kzg_abi.}
 
-proc blob_to_kzg_commitment*(res: var KzgCommitment,
-                         blob: KzgBlob,
+proc blob_to_kzg_commitment*(res: ptr KzgCommitment,
```

Comment:
we should be able to keep the `var` here, no? given this is an "out" argument required to be non-null..

Reply:
thanks, restored.